src/helpers/grpcurl.js

```diff
@@ -0,0 +1,35 @@
+export function formatGrpcUrl(
+    selectedServer,
+    selectedService,
+    selectedMethod,
+    metaData,
+    requestText
+) {
+    // trim whitespace at end of lines
+    requestText = requestText?.replaceAll(/(?:[ \t]+)(\n)/g, '$1') || '';
+
+    var headers = '';
```

Comment:
Can you use `let` instead of `var`

Reply:
@robrichard yes. Also, let me know if there is a more idiomatic way to iterate over the json entries, build the header string, and interpolate it into the grpcurl string below

Reply:
the rest of this looks good